lv_objx/lv_chart.c

```diff
@@ -819,4 +821,70 @@ static void lv_chart_draw_vertical_lines(lv_obj_t * chart, const lv_area_t * mas
         }
     }
 }
+
+/**
+ * Draw the data lines as areas on a chart
+ * @param obj pointer to chart object
+ */
+static void lv_chart_draw_areas(lv_obj_t * chart, const lv_area_t * mask)
+{
+	lv_chart_ext_t * ext = lv_obj_get_ext_attr(chart);
+
+	uint16_t i;
+	lv_point_t p1;
+	lv_point_t p2;
+	lv_coord_t w = lv_obj_get_width(chart);
+	lv_coord_t h = lv_obj_get_height(chart);
+	lv_coord_t x_ofs = chart->coords.x1;
+	lv_coord_t y_ofs = chart->coords.y1;
+	int32_t y_tmp;
+	lv_coord_t p_prev;
+	lv_coord_t p_act;
+	lv_chart_series_t * ser;
+	lv_opa_t opa_scale = lv_obj_get_opa_scale(chart);
```

Comment:
@kisvegabor Does `lv_draw_triangle` support the `opa_scale` property? If not would it be simple for me to add?

Reply:
@embeddedt 
Thank you, but once I was there I added it. Also removed the `define`s to enable triangle drawing. (I used them when played 3D engines)

Reply:
@kisvegabor No problem; I just wanted to offload some of the work from you. You do a lot of work for this project! :smiley:

Reply:
@embeddedt Thank you! It's really tough now. I believe in with v6.0 we will have almost all most-wanted features so I count on less develepment after it. :)